pkg/psmdb/pmm.go

```diff
@@ -7,13 +7,13 @@ import (
 	api "github.com/percona/percona-server-mongodb-operator/pkg/apis/psmdb/v1"
 )
 
-const (
-	PMMUserKey     = "PMM_SERVER_USER"
-	PMMPasswordKey = "PMM_SERVER_PASSWORD"
-)
-
 // PMMContainer returns a pmm container from given spec
-func PMMContainer(cr *api.PerconaServerMongoDB, secrets string, customLogin bool, clusterName string, v120OrGreater bool, v160OrGreater bool, customAdminParams string) corev1.Container {
+func PMMContainer(cr *api.PerconaServerMongoDB, secret *corev1.Secret, customAdminParams string) corev1.Container {
+	_, oka := secret.Data[api.PMMAPIKey]
+	_, okl := secret.Data[api.PMMUserKey]
+	_, okp := secret.Data[api.PMMPasswordKey]
+	customLogin := oka || (okl && okp)
```

Comment:
You could move this var closer to where it is first used, which is above line 119.